styles/Vocab/Plone/accept.txt

```diff
@@ -0,0 +1,10 @@
+`plone.api`
```

Comment:
The only entries here should be local to Volto docs. Please verify that all these words do in fact exist in the Volto docs, and remove those that do not exist.